atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransactionManager.java

```diff
@@ -69,6 +66,31 @@ public LockService getLockService() {
         public void registerClosingCallback(Runnable closingCallback) {
             manager.registerClosingCallback(closingCallback);
         }
+
+        @Override
+        public boolean isInitialized() {
+            if (!isInitialized) {
+                tryInitialize();
+            }
+
+            return isInitialized;
+        }
+
+        private synchronized void tryInitialize() {
```

Comment:
Rename to `checkInitialized`? We are not actually doing the initialization here, just checking if it was done in the background.